tst/identity.json

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "construct": "tst/in/identity.evm",
+        "tests": [
+            {
+                "debug": "0x60",
+                "gasUsed": "0xbe07",
```

Comment:
This is [off](https://holesky.etherscan.io/vmtrace?txhash=0x34b8dbb8024ef666f4b9b3a0630ca51997033680623d66887052c2c3570a3347&type=gethtrace2) by `27482`.

Reply:
2500 is (G_COLD_ACCOUNT - G_ACCESS) and 25000 is G_NEWACCOUNT. Still off by 18.